db/migrate/20230418133204_change_xref_cas_datatype.rb

```diff
@@ -0,0 +1,10 @@
+class ChangeXrefCasDatatype < ActiveRecord::Migration[6.1]
+  def change
+    Sample.find_each do |sample|
+      if sample.xref['cas']
```

Comment:
There is no default value for xref set in the db. Some entries can have nil value for xref.